packages/design-tokens/src/types.ts

```diff
@@ -20,6 +20,9 @@ type Hex = string
 // Once TypeScript is upgraded in the repo, you can use Lowercase<>
 // type Hex = Lowercase<string>
 
+/**
+ * @deprecated Not needed if you are using `KaizenProvider` from `@kaizen/components` or `defaultPreset` from next-services.
+ */
 export type TypographyFont = {
```

Comment:
Most of these are duplicated in `js/types.ts` and exported. TS conflict, so only keep one of each (or don't export in one file).

Reply:
Shouldn't clash, cause they're not being exported from the same index?

Reply:
Ohh interesting. Yeah sorry was looking at the wrong barrel file and thought they were being exported together.
I guess it's fine? But given the ones that are exported at the root are the deprecated ones, would that cause confusion?

Reply:
Mmm risk I'm willing to take haha